contracts/Compass.vy

```diff
@@ -250,3 +317,107 @@ def deploy_erc20(_paloma_denom: String[64], _name: String[64], _symbol: String[3
     event_id: uint256 = unsafe_add(self.last_event_id, 1)
     self.last_event_id = event_id
     log ERC20DeployedEvent(_paloma_denom, erc20, _name, _symbol, _decimals, event_id)
+
+@external
+@view
+def arbitrary_view(contract_address: address, payload: Bytes[1024]) -> Bytes[1024]:
+    return raw_call(contract_address, payload, is_static_call=True, max_outsize=1024)
```

Comment:
This can be removed, I thought we needed it, but we don't. Thanks for your efforts.